hubmap_api_py_client/external.py

```diff
@@ -275,17 +283,23 @@ def _add_method(output_type, ResultsSetSubclass, args_type, doc):
         '''
     ),
     'dataset': (
-        'where_has',
+        'min_cells',
         '''
         Select a set of datasets. If no params are provided, selects the set of all datasets.
         Otherwise, selects a set of datasets filtered based on parameters supplied.
 
         Args:
             where (str): The type of entity for which identifiers are supplied as input to query.
                 Must be one of ["dataset", "cell", "cluster"].
-            has (List[str]): A list of entity identifiers (dataset_uuids, etc) supplied as input to
-                the query.
-
+            has (List[str]): A list of entity identifiers or expressions (dataset_uuids, etc) supplied as
+                input to the query.
+            genomic_modality (str): Modality to consider in quantitative queries.
+                Required for queries in which "where" is "gene". Must be one of ["rna", "atac"].
+            logical_operator (str): The logical operator applied to filters generated by elements of
+                "has". Required for queries in which "where" is "gene" or "protein" and "has" contains
+                more than one element. Must be one of ["and", "or"].
```

Comment:
Could the same results be obtained by using set operations on the results? We have a really complex interface here, and pretty thin usage across it, and the more features that get added, the more I worry about bugs we just haven't hit yet.

I'd prefer that it just does `and`, and that this not be exposed as a kwarg.

Reply:
The idea I'd had was that the gene/protein->dataset queries would take the same parameters as the gene/protein->cell queries, plus the min_cell_percentage, but point taken.  I will make that change.